integration-libs/opps/root/login-required/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export * from './opps-login-required.module';
```

Comment:
For extensibility purposes, I believe we want to also export the `guard` file

Reply:
done